packages/@okta/vuepress-site/use_cases/event_hooks/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Event Hooks
+excerpt: Use Okta events to drive custom process flows.
+---
+
+# Event Hooks
+
+<ApiLifecycle access="ea" />
+
+## What Are Okta Event Hooks?
+
+Event hooks are outbound calls from Okta, sent when specified events occur in your org. They take the form of HTTPS REST calls to a URL you specify, encapsulating information about the events in JSON objects in the request body. These calls from Okta are meant to be used as triggers for process flows within your own software systems.
+
+To handle event hook calls from Okta, you need to implement a web service with an Internet-accessible endpoint. It's your responsibility to develop the code and to arrange its hosting on a system external to Okta. Okta defines the REST API contract for the requests that it will send.
+
+Event hooks are related to, but different from, Okta [inline hooks](/use_cases/inline_hooks/). Event hooks are meant to provide information about events that occurred, not offer a way to affect the underlying Okta process flow. For that, you need to use inline hooks. Also, unlike inline hooks, event hooks are asynchronous calls, meaning that the process flow that triggered the event hook continues without stopping or waiting for any response from your external service.
+
+## Which Events are Eligible?
+
+During the initial configuration procedure for an event hook, you specify which event types you want the event hook to deliver. The event types that can be specified are a subset of the event types that the Okta System Log captures. To see the list of event types currently eligible for use in event hooks, query the Event Types catalog with the query parameter `event-hook-eligible`:
+
+[https://developer.okta.com/docs/api/resources/event-types/?q=event-hook-eligible](/docs/api/resources/event-types/?q=event-hook-eligible)
+
+## Requests Sent by Okta
+
+The requests sent from Okta to your external service are HTTPS requests. POST requests are used for the ongoing delivery of events, and a one-time GET request is used for verifying your endpoint.
+
+### One-Time Verification Request
```

Comment:
In addition - it may take upto a minute for the event hook to used by Okta after it has been verified.

Reply:
Added a sentence: "After your event hook is verified, it can take up to a minute for delivery of events to your endpoint to begin." (All changes are in new pull request: https://github.com/okta/okta-developer-docs/pull/227)